src/main/java/generator/ClientGenerator.java

```diff
@@ -0,0 +1,53 @@
+package generator;
+
+import models.Client;
+import models.Phone;
+import java.util.ArrayList;
+import java.util.List;
+
+public final class ClientGenerator {
+    public static final List<Client> clients = List.of(
+            new Client(0, "Alice", 777, new ArrayList<>(List.of(
+                    new Phone("+123456789", "Mobile"),
+                    new Phone("+4355762", "Mobile"),
+                    new Phone("+23464765", "Wired")
+            ))),
+            new Client(1, "Bob", 23, new ArrayList<>(List.of(
+                    new Phone("+4556754743", "Mobile")
+            ))),
+            new Client(2, "Clint", 54, new ArrayList<>(List.of(
+                    new Phone("+5778786", "Wired"),
+                    new Phone("+12435632", "Wired")
+            ))),
+            new Client(3, "David", 90, new ArrayList<>(List.of(
+                    new Phone("+45876221", "Mobile"),
+                    new Phone("+3435535", "Mobile"),
+                    new Phone("+5768687", "Wired"),
+                    new Phone("+90909009", "Mobile")
+            ))),
+            new Client(4, "Ernest", 0, new ArrayList<>(List.of(
+
+            ))),
+            new Client(5, "Falcon", 43, new ArrayList<>(List.of(
+                    new Phone("+2223", "Mobile"),
+                    new Phone("+222", "Wired")
+            ))),
+            new Client(6, "Gorge", 54, new ArrayList<>(List.of(
+
+            ))),
+            new Client(7, "Alice", 90, new ArrayList<>(List.of(
+                    new Phone("+55789", "Wired")
+            ))),
+            new Client(8, "Bob", 5455, new ArrayList<>(List.of(
+                    new Phone("+1111", "Wired"),
+                    new Phone("+11212", "Wired")
+            ))),
+            new Client(9, "Clint", 111, new ArrayList<>(List.of(
+                    new Phone("+421113423", "Mobile")
+            )))
+    );
+
+    private ClientGenerator() {
+
+    }
```

Comment:
why do you need private constructor, final class and static method?

Reply:
1) Private constructor does not allow to create instances of the class
2) Final class, because I don't want this class to be extended by any class
3) I only need take List of clients from this class, so i don't want to create instance for this simple task

Reply:
it means this is a utility class.
To reduce this bolilerplate code you can use this annotation @UtilityClass: https://projectlombok.org/features/experimental/UtilityClass